src/datagen/generated/minecolonies/data/minecolonies/quests/tutorial/builder.json

```diff
@@ -0,0 +1,283 @@
+          }
+        },
+        {
+          "answer": "minecolonies.quests.tutorial.builder.obj2.answer1",
+          "result": {
+            "type": "minecolonies:dialogue",
+            "options": [
+              {
+                "answer": "minecolonies.quests.tutorial.builder.obj2.answer1.reply.answer0",
+                "result": {
+                  "type": "minecolonies:dialogue",
+                  "options": [
+                    {
+                      "answer": "minecolonies.quests.tutorial.builder.obj2.answer1.reply.answer0.reply.answer0",
+                      "result": {
+                        "type": "minecolonies:advanceobjective",
+                        "go-to": 3
+                      }
+                    }
+                  ],
+                  "text": "minecolonies.quests.tutorial.builder.obj2.answer1.reply.answer0.reply"
+                }
+              }
+            ],
+            "text": "minecolonies.quests.tutorial.builder.obj2.answer1.reply"
+          }
+        },
+        {
+          "answer": "minecolonies.quests.tutorial.builder.obj2.answer2",
+          "result": {
+            "type": "minecolonies:dialogue",
+            "options": [
+              {
+                "answer": "minecolonies.quests.tutorial.builder.obj2.answer2.reply.answer0",
+                "result": {
+                  "type": "minecolonies:advanceobjective",
+                  "go-to": 5
+                }
+              }
+            ],
+            "text": "minecolonies.quests.tutorial.builder.obj2.answer2.reply"
+          }
+        },
+        {
+          "answer": "minecolonies.quests.tutorial.builder.obj2.answer3",
+          "result": {
+            "type": "minecolonies:return"
+          }
+        }
+      ],
+      "target": 0,
+      "text": "minecolonies.quests.tutorial.builder.obj2"
+    },
+    {
+      "type": "minecolonies:buildbuilding",
+      "details": {
+        "type": "minecolonies:builder",
+        "count-existing": true,
+        "lvl": 1,
+        "next-objective": 4,
+        "qty": 1,
+        "target": 0
+      },
+      "unlocks-rewards": [
+        0
+      ]
+    },
+    {
+      "type": "minecolonies:dialogue",
+      "options": [
+        {
+          "answer": "minecolonies.quests.tutorial.builder.obj4.answer0",
+          "result": {
+            "type": "minecolonies:advanceobjective",
+            "go-to": 5
+          }
+        }
+      ],
+      "target": 0,
+      "text": "minecolonies.quests.tutorial.builder.obj4"
+    }
+  ],
+  "parents": [
+    "minecolonies:tutorial/welcome"
+  ],
+  "rewards": [
+    {
+      "type": "minecolonies:item",
+      "details": {
+        "item": "minecraft:cobblestone",
+        "qty": 32
+      }
+    }
+  ],
+  "triggerOrder": "1 || 2",
+  "triggers": [
+    {
+      "type": "minecolonies:citizen",
+      "state": {
+        "match": {
+          "finpartquests": [
```

Comment:
what does "finpartquests" mean? is it a typo?

Reply:
nbt tag for finished quest participation